Formula/sshguard.rb

```diff
@@ -16,12 +16,22 @@ class Sshguard < Formula
   depends_on "autoconf" => :build
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
-                          "--with-firewall=#{firewall}"
+                          "--sysconfdir=#{etc}"
     system "make", "install"
+    mkdir "etc"
+    cp "examples/sshguard.conf.sample", "etc/sshguard.conf"
+    inreplace "etc/sshguard.conf" do |s|
+      s.gsub! /^#BACKEND=.*$/, "BACKEND=\"#{libexec}/sshg-fw-#{firewall}\""
+      if MacOS.version >= :sierra
+        s.gsub! %r{^#LOGREADER="/usr\/bin\/log}, "LOGREADER=\"/usr/bin/log"
+      else
+        s.gsub! /^#FILES.*$/, "FILES=#{log_path}"
+      end
+    end
+    etc.install "etc/sshguard.conf"
```

Comment:
So unless you have a suggestion for how to deal with etc/"sshguard.conf" referring to the pre-existing version, I think making a copy in the build tree and etc.install'ing it is the way to go. I've changed it in my version to make the copy in the examples/ directory instead of creating an etc/ because there is no point to that now that I'm pointing the binary at the proper etc directory.